scripts/netlify.js

```diff
@@ -0,0 +1,10 @@
+const shell = require('shelljs');
+
+shell.exec('git clone https://github.com/videojs/http-streaming');
+shell.cd('http-streaming');
+shell.exec('npm ci');
+shell.exec('npm link ../');
+shell.exec('npm run netlify');
```

Comment:
can we add a comment to what this is doing? Specifically, that's it's running the netlify build of VHS itself.